api/v1alpha1/instancegroup_types.go

```diff
@@ -173,6 +182,29 @@ type EKSManagedConfiguration struct {
 	Version            string              `json:"version,omitempty"`
 }
 
+// EKSCFConfiguration defines the context of an AWS Instance Group using EKSCF
+type EKSCFConfiguration struct {
```

Comment:
This one was also removed in master (EKSCF deprecation)

Reply:
done